stix_shifter_modules/sentinelone/stix_translation/json/to_stix_map.json

```diff
@@ -0,0 +1,645 @@
+    "key": "process.extensions.x-sentinelone-process.active_content_hash",
+    "object": "process"
+  },
+  "srcProcActiveContentSignedStatus": {
+    "key": "process.extensions.x-sentinelone-process.active_content_signed_status",
+    "object": "process"
+  },
+  "tgtProcActiveContentSignedStatus": {
+    "key": "process.extensions.x-sentinelone-process.active_content_signed_status",
+    "object": "process"
+  },
+  "srcProcRpid": {
+    "key": "process.extensions.x-sentinelone-process.rpid",
+    "object": "process"
+  },
+  "srcProcTid": {
+    "key": "process.extensions.x-sentinelone-process.tid",
+    "object": "process"
+  },
+  "srcProcParentStartTime": {
+    "key": "process.extensions.x-sentinelone-process.parent_process_start_time",
+    "object": "process"
+  },
+  "srcProcParentImagePath": {
+    "key": "process.extensions.x-sentinelone-process.parent_image_path",
+    "object": "process"
+  },
+  "srcProcParentUid": {
+    "key": "process.extensions.x-sentinelone-process.parent_uid",
+    "object": "process"
+  },
+  "srcProcParentImageSha1": {
+    "key": "process.extensions.x-sentinelone-process.parent_sha1_image",
+    "object": "process"
+  },
+  "srcProcParentImageSha256": {
+    "key": "process.extensions.x-sentinelone-process.parent_sha256_image",
+    "object": "process"
+  },
+  "srcProcParentImageMd5": {
+    "key": "process.extensions.x-sentinelone-process.parent_md5_image",
+    "object": "process"
+  },
+  "netEventDirection": {
+    "key": "network-traffic.extensions.x-sentinelone-network-action.event_direction",
+    "object": "nt"
+  },
+  "netConnStatus": {
+    "key": "network-traffic.extensions.x-sentinelone-network-action.connection_status",
+    "object": "nt"
+  },
+  "loginsBaseType": {
+    "key": "user-account.extensions.x-sentinelone-login.base_type",
+    "object": "user"
+  },
+  "srcMachineIP": {
+    "key": "user-account.extensions.x-sentinelone-login.machine_ip",
+    "object": "user"
+  },
+  "loginType": {
+    "key": "user-account.extensions.x-sentinelone-login.login_type",
+    "object": "user"
+  },
+  "loginFailureReason": {
+    "key": "user-account.extensions.x-sentinelone-login.login_failure_reason",
+    "object": "user"
+  },
+  "loginAccountDomain": {
+    "key": "user-account.extensions.x-sentinelone-login.account_domain",
+    "object": "user"
+  },
+  "loginSessionId": {
+    "key": "user-account.extensions.x-sentinelone-login.session_id",
+    "object": "user"
+  },
+  "tgtFileType": {
+    "key": "file.extensions.x-sentinelone-file.file_type",
+    "object": "file"
+  },
+  "tgtFileExtension": {
+    "key": "file.extensions.x-sentinelone-file.extention",
+    "object": "file"
+  },
+  "tgtFileId": {
+    "key": "file.extensions.x-sentinelone-file.file_id",
+    "object": "file"
+  },
+  "tgtFileDescription": {
+    "key": "file.extensions.x-sentinelone-file.description",
+    "object": "file"
+  },
+  "tgtFileLocation": {
+    "key": "file.extensions.x-sentinelone-file.location",
```

Comment:
Is this a geolocation or the file path?

Reply:
It is not a geo location or file path, it is a Enum field contains value "Local","Network Drive", "External Device". Hence we did not change it.